### DISCUSSION TITLE
this is what i got when trying 2 install & start

### DISCUSSION THREAD

**Discussion by joe-bro-1** [4ian/GDevelop#4790] [Archive] [answered]
## don’t know if this is an error on my end
<img width="1436" alt="Screenshot 2023-01-05 at 3 47 58 PM" src="https://user-images.githubusercontent.com/117040706/210878126-f83e7c37-293b-4770-8343-bef3857c0e8c.png">

<img width="1436" alt="Screenshot 2023-01-05 at 3 46 55 PM" src="https://user-images.githubusercontent.com/117040706/210877896-19af3898-29b9-42a3-941d-b553717817ab.png">
<img width="1436" alt="Screenshot 2023-01-05 at 3 47 09 PM" src="https://user-images.githubusercontent.com/117040706/210877922-eea5f9ee-b2e3-46fe-9893-e41a4fdcab8e.png">
<img width="1436" alt="Screenshot 2023-01-05 at 3 47 16 PM" src="https://user-images.githubusercontent.com/117040706/210877965-ae3fe476-4e7a-4612-ac4b-0959a7c2bd83.png">
<img width="1436" alt="Screenshot 2023-01-05 at 3 47 25 PM" src="https://user-images.githubusercontent.com/117040706/210878014-2683398a-1a91-4520-9e48-852b3e18b261.png">
<img width="1436" alt="Screenshot 2023-01-05 at 3 47 32 PM" src="https://user-images.githubusercontent.com/117040706/210878040-f1d06e9e-d247-4563-adb5-3ae3d31626ff.png">
<img width="1436" alt="Screenshot 2023-01-05 at 3 47 38 PM" src="https://user-images.githubusercontent.com/117040706/210878061-c37a5eb1-f7e6-4530-898d-6048f0ff4a53.png">
<img width="1436" alt="Screenshot 2023-01-05 at 3 47 44 PM" src="https://user-images.githubusercontent.com/117040706/210878085-b9905188-706f-42b2-b3b7-07f6fe3c46aa.png">
<img width="1436" alt="Screenshot 2023-01-05 at 3 47 52 PM" src="https://user-images.githubusercontent.com/117040706/210878104-ec84c86c-a3d1-4631-9765-2e812f04cb1d.png">

**Comment by 4ian** [comment, marked as answer]
See the workaround here: https://github.com/4ian/GDevelop/issues/4756 :)
